src/index/inverted_index_reader.rs

```diff
@@ -315,6 +330,60 @@ impl InvertedIndexReader {
         Ok(true)
     }
 
+    /// Warmup a block postings given a range of `Term`s.
+    /// This method is for an advanced usage only.
+    ///
+    /// returns a boolean, whether a term matching the range was found in the dictionary
+    pub async fn warm_postings_automaton<A: Automaton + Clone>(
+        &self,
+        automaton: A,
+        // with_positions: bool, at the moment we have no use for it, and supporting it would add
+        // complexity to the coalesce
+    ) -> io::Result<bool>
+    where
+        A::State: Clone,
+    {
+        // merge holes under 4MiB, that's how many bytes we can hope to receive during a TTFB from
+        // S3 (~80MiB/s, and 50ms latency)
+        let merge_holes_under = (80 * 1024 * 1024 * 50) / 1000;
```

Comment:
```suggestion
        let merge_holes_under = (80 * 1024 * 1024 * 50) / 1000;
```

merge_holes_under_bytes